.github/workflows/package-apisix-openresty-rpm-el7.yml

```diff
@@ -21,8 +21,6 @@ jobs:
       - name: install dependencies
         run: |
           sudo apt-get install -y make ruby ruby-dev rubygems build-essential
-          sudo gem install --no-document fpm
-          sudo apt-get install -y rpm
```

Comment:
@EnableAsync any reason to remove these two lines?

Reply:
Now that fpm is installed in docker, there is no need to install it in github action.

Reply:
@EnableAsync Got it. Thanks.